src/app/parcels/ExpandedParcelDetails.tsx

```diff
@@ -59,7 +67,13 @@ const ExpandedParcelDetailsView = ({ parcelId }: Props): ReactElement => {
 
             {parcelDetails && (
                 <>
-                    <DataViewer data={parcelDetails.expandedParcelData} />
+                    {!parcelDetails.expandedParcelData.isActive && (
```

Comment:
I've fixed this now - the only case where clientIsActive could be undefined was when the parcel had no associated client. I  changed the definition of clientIsActive to use client rather than rawParcelDetails.client, because we check for client being null before the declaration of clientIsActive, and return from the function if it is null. Now typescript knows that clientIsActive is a boolean only.

Reply:
Thanks for looking into this! Happy with that solution :)